base-migrate-app/src/app/migrate/page.tsx

```diff
@@ -38,16 +48,54 @@ function MigratePage() {
 
   const handleSubmit = (e: any) => {
     e.preventDefault();
+
+    if (!cookies?.authtoken) {
+      return navigate.push('/home');
+    }
+
+    nextStep();
+
     deployToken(formData.token_address, formData.token_name, formData.token_symbol);
   };
-  console.log(isConfirmed, isPending);
 
-  React.useEffect(() => {
-    const fetchData = async () => {
+  const fetchData = async () => {
+    try {
+      if (isPending || !isConfirmed) return;
+
       const data = await getTransactionData();
       console.log(data, 'tx data');
       console.log('deployed token on base:', data?.logs[0].address);
-    };
+
+      const body = {
+        chainId,
+        logoUrl: formData.logo,
+        tokenData: {
+          name: formData.token_name,
+          symbol: formData.token_symbol,
+          decimals: formData.token_decimal,
+          description: formData.token_description,
+          website: formData.website,
+          twitter: formData.twitter,
+          tokens: {
+            ethereum: {
+              address: formData.token_address,
+            },
+            base: {
```

Comment:
Added support for `sepolia`  and `base-sepolia` but you can add that only if the user is connected to testnet